cl/opinion_page/forms.py

```diff
@@ -212,15 +282,42 @@ def __init__(self, *args, **kwargs) -> None:
                 choices=[("TN WC", "TN WC")],
                 attrs={"class": "form-control"},
             )
-            del self.fields["second_judge"]
-            del self.fields["third_judge"]
+            self.drop_fields(
+                [
+                    "date_argued",
+                    "date_reargued",
+                    "panel",
+                    "second_judge",
+                    "third_judge",
+                ]
+            )
+
+        elif self.pk == "me":
+            self.fields["cite_reporter"].widget = forms.Select(
+                choices=[("ME", "ME")],
+                attrs={"class": "form-control"},
+            )
+            self.drop_fields(["lead_author", "second_judge", "third_judge"])
         else:
```

Comment:
It's worth being explicit in this `else` statement since things are getting complex. I'd suggest making it:

```
elif self.pk == "xyz'
    do_stuff
else:
    Raise_an_error or throw a 500 or something with an "unknown court" error.
```

Reply:
√